File: sample/java/app/src/main/java/com/igaworks/adbrixjavasample/BaseApplication.java
package com.igaworks.adbrixjavasample;

import android.app.Application;
import android.util.Log;

import com.igaworks.adbrix.Adbrix;
import com.igaworks.adbrix.AdbrixConfig;

public class BaseApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        AdbrixConfig.Builder config = new AdbrixConfig.Builder()
            .setLogEnable(true)
            .setLogLevel(Log.VERBOSE)
            .setCollectGoogleAdvertisingId(true);
        Adbrix.getInstance().init(this, BuildConfig.ADBRIX_APPLICATION_KEY, BuildConfig.ADBRIX_SECRET_KEY, config.build());
    }
}

File: sample/kotlin/app/src/main/java/com/igaworks/adbrixkotlinsample/BaseApplication.kt
package com.igaworks.adbrixkotlinsample

import android.app.Application
import android.util.Log
import com.igaworks.adbrix.Adbrix
import com.igaworks.adbrix.AdbrixConfig

class BaseApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        val config = AdbrixConfig.Builder()
            .setLogEnable(true)
            .setLogLevel(Log.VERBOSE)
            .setCollectGoogleAdvertisingId(true)
        Adbrix.getInstance().init(
            this,
            BuildConfig.ADBRIX_APPLICATION_KEY,
            BuildConfig.ADBRIX_SECRET_KEY,
            config.build()
        )
    }
}